client/my-sites/checkout/checkout/index.jsx

```diff
@@ -474,6 +474,11 @@ export class Checkout extends React.Component {
 	}
 
 	maybeRedirectToConciergeNudge( pendingOrReceiptId ) {
+		// For testing Frankenflow we disable any redirect to Nudge
```

Comment:
It's a valid feature on its own for some flows — maybe we can even leave the comment out or make it more generic? :-)

Reply:
Good point! So we'll enable it in `CheckoutContainer` based on `isComingFromFrankenflow` flag (which will be used also for customising the checkout screen header) and other flows could do the same.